tools/create/scaffold/scaffold.go

```diff
@@ -0,0 +1,76 @@
+package scaffold
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func RunE(cmd *cobra.Command, args []string) error {
```

Comment:
`RunE` is a very peculiar function name 😁 
Also args is unused here so you can use blank identifier `_`